docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -72,6 +76,12 @@ export default class DialogPage extends React.Component {
             desc: 'Set to true to have the dialog automatically open on mount.',
```

Comment:
I presume that we should also put back the `onShow`, `onDismiss`, `modal`, `show()` and `dismiss()` with a deprecation warning then?